src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -655,7 +655,7 @@ class AddItemsPage extends Component {
     const file = event.target.files[0];
     const { stockMovementId } = this.state.values;
 
-    formData.append('importFile', file);
+    formData.append('importFile', file.slice(0, file.size, 'text/csv'));
```

Comment:
There is a filter in the file input, so you can choose only files with .csv extension, also this mime type issue is not that simple, because it will not always be "application/vnd.ms-excel", it will have this type only on Window that have Excel installed, I think it has something to do with file extension association to the program. If there is not Excel installed the mime type will be "application/octet-stream", which can be many types of files.

Reply:
Good point. But the problem with filtering only text/csv is that PIH will never see any files. We need to find a workaround that allows Windows users to "Save as ..." their Excel CSV to get them to write out as text/csv, right? Or right click and set the mime type in the file properties (if that's even still a thing you can do).

Reply:
As far as I know the mime type is not a property of the file, it's more like a table stored in the system which is mapping file extension to mime type and the issue is one the Windows there is no mapping for .csv by default, if you install Excel it is adding mapping to "application/vnd.ms-excel". Of course you can fix it by editing the Windows registry and adding the missing mime type, but I don't think users will want to do this.

Reply:
My point is that when a Windows users goes to import a file, they won't see any CSV files because they are all mapped as application/vnd.ms-excel. So while this fixes the issue of not allowing non-text-csv files to be uploaded, it also prevents the Windows user from uploading any files because none of their CSV file will show up in the File Chooser. Right?

Reply:
Actually the file chooser is filtering files by extension, not the mime type (because it's not a property of a file), so users on all operating systems will see only the csv files in the file chooser.

Reply:
I get that it's not a property of the file but the code is definitely filtering on mime type. "text/csv" is a mime-type and ".csv" is an extension. My assumption was that the File Chooser was "decorating" all files with a mime-type before filtering. For the record, I'm happy with this PR either way. I'm just trying to figure out if it's the entire solution or just part of the solution.

Reply:
Ah ok that's what I was missing. Thanks for including the line that references the ".csv" type. Makes more sense now.

Reply:
This mime type decoration is done by the browser and the browser is using system setting to get the mime type for given extension, also the type it's read only, that's why in this PR I'm copying the File to new Blob (it's parent "class" of the File) which allows me to set the type manually.